### PR TITLE
[tests-only] Fix: Delete file in trashbin UI test step not working #4437

### LINKED ISSUE #4437: Delete file in trashbin UI test step not working
The `webUITrashbinDelete` test suite has not been running in CI. That is fixed by PR #4435 

But one scenario fails: https://drone.owncloud.com/owncloud/phoenix/12088/76/11
```
Failures:

1) Scenario: Delete single file from deleted files list (attempt 2) # tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:112
   ✔ Before # tests/acceptance/setup.js:28
   ✔ Before # tests/acceptance/setup.js:32
   ✔ Before # tests/acceptance/setup.js:36
   ✔ Before # tests/acceptance/setup.js:46
   ✔ Before # tests/acceptance/setup.js:68
   ✔ Before # tests/acceptance/setup.js:75
   ✔ Before # tests/acceptance/stepDefinitions/filesContext.js:16
   ✔ Before # tests/acceptance/stepDefinitions/generalContext.js:242
   ✔ Before # tests/acceptance/stepDefinitions/generalContext.js:281
   ✔ Given user "user1" has been created with default attributes # tests/acceptance/stepDefinitions/provisioningContext.js:152
   ✔ And user "user1" has created file "sample,1.txt" # tests/acceptance/stepDefinitions/filesContext.js:876
   ✔ And user "user1" has created folder "Folder,With,Comma" # tests/acceptance/stepDefinitions/filesContext.js:852
   ✔ And user "user1" has logged in using the webUI # tests/acceptance/stepDefinitions/loginContext.js:67
   ✔ And the following files have been deleted by user "user1" # tests/acceptance/stepDefinitions/filesContext.js:204
       | name              |
       | data.zip          |
       | lorem.txt         |
       | lorem-big.txt     |
       | simple-folder     |
       | sample,1.txt      |
       | Folder,With,Comma |
   ✔ And the user has browsed to the trashbin page # tests/acceptance/stepDefinitions/filesContext.js:56
   ✖ When the user deletes the file "lorem.txt" from the deleted files list # tests/acceptance/stepDefinitions/filesContext.js:929
       TypeError: Cannot read property 'selector' of undefined
           at Page.getActionSelector (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:28:74)
           at Page.performFileAction (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:41:47)
           at Page._callee6$ (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:133:12)
           at tryCatch (/var/www/owncloud/phoenix/node_modules/regenerator-runtime/runtime.js:63:40)
           at Generator.invoke [as _invoke] (/var/www/owncloud/phoenix/node_modules/regenerator-runtime/runtime.js:293:22)
           at Generator.next (/var/www/owncloud/phoenix/node_modules/regenerator-runtime/runtime.js:118:21)
           at asyncGeneratorStep (/var/www/owncloud/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
           at _next (/var/www/owncloud/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
           at /var/www/owncloud/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
           at new Promise (<anonymous>)
           at Page.<anonymous> (/var/www/owncloud/phoenix/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12)
           at Page.deleteResourceImmediately (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:132:30)
           at Proxy._callee8$ (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/FilesPageElement/filesList.js:86:52)
           at tryCatch (/var/www/owncloud/phoenix/node_modules/regenerator-runtime/runtime.js:63:40)
           at Generator.invoke [as _invoke] (/var/www/owncloud/phoenix/node_modules/regenerator-runtime/runtime.js:293:22)
           at Generator.next (/var/www/owncloud/phoenix/node_modules/regenerator-runtime/runtime.js:118:21)
   - Then file "lorem.txt" should not be listed on the webUI # tests/acceptance/stepDefinitions/filesContext.js:403
   - When the user reloads the current page of the webUI # tests/acceptance/stepDefinitions/filesContext.js:474
   - Then file "lorem.txt" should not be listed on the webUI # tests/acceptance/stepDefinitions/filesContext.js:403
   ✔ After # tests/acceptance/stepDefinitions/provisioningContext.js:226
   ✔ After # tests/acceptance/stepDefinitions/generalContext.js:293
   ✔ After # tests/acceptance/stepDefinitions/generalContext.js:266
   ✔ After # tests/acceptance/setup.js:110
   ✔ After # tests/acceptance/setup.js:106
   ✔ After # tests/acceptance/setup.js:102
   ✔ After # tests/acceptance/setup.js:95
   ✔ After # tests/acceptance/setup.js:86
   ✔ After # tests/acceptance/setup.js:54
```

Something goes wrong with the step:
```
    When the user deletes the file "file-name" from the deleted files list
```

The scenario has been skipped.

Investigate and fix.

### PR DESCRIPTION
**Pull request by saw-jan**
## Related Issue

fixes issue #4437

## How Has This Been Tested?

- local environment

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by swoichha — COMMENTED** [review]
LGTM

File: tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ module.exports = {
       selector: '//button[@aria-label="Rename"]',
       locateStrategy: 'xpath'
     },
-    deleteImmediatelyButtonInFileRow: {
+    deleteImmediatelyButtonInAccordion: {
```
Comment (jasson99) [inline tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:189]: This element is same as the `deleteButtonInAccordion` element in line number 165. May be we can think of a way about  using only one of them. For that the FileAction should be checked. @individual-it what do you think about this?

Comment (saw-jan) [reply tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:189]: we can also change
`this.performFileAction(this.FileAction.deleteImmediately)`
to 
`this.performFileAction(this.FileAction.delete)`

also clean the `deleteImmediately` occurances

Comment (phil-davis) [reply tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js:189]: We can do that in a different PR. For now it is good to get the test scenario enabled again.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**